randovania/games/planets_zebeth/pickup_database/pickup-database.json

```diff
@@ -0,0 +1,298 @@
+                "a ",
+                "missile-related upgrade"
+            ],
+            "hinted_as_major": false
+        },
+        "life_support": {
+            "long_name": "Life Support",
+            "hint_details": [
+                "a ",
+                "life support system"
+            ],
+            "hinted_as_major": false
+        }
+    },
+    "standard_pickups": {
+        "Bombs": {
+            "pickup_category": "morph_ball",
+            "broad_category": "morph_ball_related",
+            "model_name": "spr_ITEM_Bomb",
+            "offworld_models": {
+                "prime1": "Morph Ball Bomb",
+                "prime2": "MorphBallBomb",
+                "dread": "powerup_bomb",
+                "am2r": "sItemBomb"
+            },
+            "progression": [
+                "Bombs"
+            ],
+            "default_shuffled_count": 1,
+            "default_starting_count": 0,
+            "preferred_location_category": "major",
+            "original_location": 6
+        },
+        "Screw Attack": {
+            "pickup_category": "movement",
+            "broad_category": "movement",
+            "model_name": "spr_ITEM_Screw_Attack",
+            "offworld_models": {
+                "prime2": "ScrewAttack",
+                "dread": "powerup_screwattack",
+                "am2r": "sItemScrewAttack"
+            },
+            "progression": [
+                "Screw Attack"
+            ],
+            "default_shuffled_count": 1,
+            "default_starting_count": 0,
+            "preferred_location_category": "major",
+            "original_location": 23
+        },
+        "Varia Suit": {
+            "pickup_category": "suit",
+            "broad_category": "life_support",
+            "model_name": "spr_ITEM_Varia",
+            "offworld_models": {
+                "prime1": "Varia Suit",
+                "prime2": "VariaSuit INCOMPLETE",
+                "dread": "powerup_variasuit",
+                "am2r": "sItemVariaSuit"
+            },
+            "progression": [
+                "Varia Suit"
+            ],
+            "default_shuffled_count": 1,
+            "default_starting_count": 0,
+            "preferred_location_category": "major",
+            "original_location": 3
+        },
+        "Hi-Jump Boots": {
+            "pickup_category": "movement",
+            "broad_category": "movement",
+            "model_name": "spr_ITEM_Jump_Boots",
+            "offworld_models": {
+                "prime1": "Space Jump Boots",
+                "prime2": "SpaceJumpBoots",
+                "dread": "powerup_doublejump",
+                "am2r": "sItemHijump"
+            },
+            "progression": [
+                "Hi-Jump"
+            ],
+            "default_shuffled_count": 1,
+            "default_starting_count": 0,
+            "preferred_location_category": "major",
+            "original_location": 16
+        },
+        "Ice Beam": {
```

Comment:
you can now add the original location for ice beam

Reply:
gotta wait till the branch for multiple original locations is merged

Reply:
oh whoops